R/notin.R

```diff
@@ -0,0 +1,6 @@
+# Intended to be used to create %notin% operator
+notin = function(example, elements) {
+  return(!match(example, elements, nomatch = 0))
```

Comment:
We should branch on `is.character(example)` and use `chmatch` for string input

Reply:
@MichaelChirico Thank you, I just added this branching.